src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -13,6 +13,7 @@
 
     public class ScenarioRunner
     {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Code", "PS0023:Use DateTime.UtcNow or DateTimeOffset.UtcNow", Justification = "Test logging")]
```

Comment:
In this specific case (and other similar ones), I wonder if we shouldn't switch to something like this, to avoid localised output:

```c#
DateTime.UtcNow.ToString("O") // e.g. 2021-07-14T07:46:23.4588246+00:00
```

That way, e.g. if I'm sharing some logs with someone in a different time zone, they won't have to do any translation to take into account the originating time zone. Everyone will just see UTC.

Reply:
If you're sharing logs with someone else, the only thing that matters is how many seconds separate two lines, not when it was. If you're running local though, you want it to be in your local time.